google/cloud/spanner_v1/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 import pkg_resources
 
-__version__ = pkg_resources.get_distribution("google-cloud-spanner").version
+__version__: str = pkg_resources.get_distribution("google-cloud-spanner").version
```

Comment:
This annotation shouldn't be needed, since you have installed `types-setuptools` in the nox session.